all.jsonnet

```diff
@@ -47,11 +47,13 @@ local re = t.receive(commonConfig {
   hashringConfigMapName: 'hashring',
 });
 
-
 local ru = t.rule(commonConfig {
+  local cfg = self,
   replicas: 1,
   rulesConfig: [{ name: 'test', key: 'test' }],
   alertmanagersURLs: ['alertmanager:9093'],
+  reloaderImage: 'quay.io/openshift/origin-configmap-reloader:' + cfg.reloaderVersion,
+  reloaderVersion: '4.5.0',
```

Comment:
This could just be reloaderImage that includes the version?

Reply:
of course. just notice that I did not expose the `reloaderVersion` in annotation. removed `reloaderVersion`